hushline/settings.py

```diff
@@ -194,6 +194,14 @@ class ProfileForm(FlaskForm):
     )
 
 
+class UpdateBrandPrimaryColorForm(FlaskForm):
+    hex_color = StringField("HEX Color", validators=[DataRequired(), Length(min=3, max=7)])
```

Comment:
Should we use a regex validator on this field? https://stackoverflow.com/a/1636354

Reply:
Addressed by (https://github.com/scidsg/hushline/pull/573/commits/75269c70e3cbd667e271e8d7e3be12a0118ca6ed, https://github.com/scidsg/hushline/pull/573/commits/5f246a8ea6caeb6b8559f562d878e68e8ce9a417)